_data/navigation.yml

```diff
@@ -1,3 +1,17 @@
 main:
+  - title: "Services"
+    url: '/services.html'
+  - title: "Previous projects"
+    url: '/previous-projects.html'
+  - title: "About us"
+    url: '/index.html'
   - title: "Contact us"
     url: '#contact'
+docs:
+  - title: "List of projects"
+    children:
+      - title: "Project 1"
+        urL: '#project1'
+      - title: "Project 2"
+        urL: '#project2'
+    
```

Comment:
Missing empty line for the end of file. Maybe the last line is not completely empty